overlord/snapstate/snapstate.go

```diff
@@ -74,26 +74,33 @@ var ErrNothingToDo = errors.New("nothing to do")
 
 var osutilCheckFreeSpace = osutil.CheckFreeSpace
 
-type ErrInsufficientSpace struct {
+type InsufficientSpaceError struct {
```

Comment:
// InsufficientSpaceError represent an error where the is not enough disk space to perform an operation.